oaknational/Oak-Web-Application
# Fix/lesq 748/search results a11y [LESQ-748]

## Description

Music year: 2011

- update aria label for single search results 
- update aria label for drop down search results

## How to test

1. Go to https://deploy-preview-2368--oak-web-application.netlify.thenational.academy
2. Search for eg 'surds'
3. You should see the aria label of 'see unit' is eg `see unit: simplifying surds`
4. Same for lessons
5. for results with a dropdown (tier or exam board) it should say `select tier for unit: simplifying surds` 

## Screenshots

How it should now look:

Lesson:
<img width="600" alt="Screenshot 2024-04-18 at 10 54 43" src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/c9564754-8d22-49d1-872b-94b5a5b069f9">

Unit:
<img width="600" alt="Screenshot 2024-04-18 at 10 54 28" src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/ca864395-3111-4ecc-aa92-3371c4ad9b6a">

Exam board:
<img width="600" alt="Screenshot 2024-04-18 at 10 55 49" src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/038c1eaa-d900-4e9f-8806-b5d945e0e81c">

Tier:
<img width="600" alt="Screenshot 2024-04-18 at 10 54 05" src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/31b638ad-bce2-4581-a9e5-0e1aa519f7fc">
